e2e/autoscript-apps/src/authn-central-login/autoscript.ts

```diff
@@ -101,6 +100,9 @@ function autoscript() {
       )
       .subscribe({
         error: (err) => {
+          if (!auth) {
+            return;
```

Comment:
I think we should add a code comment here to remind our future selves why this is here.